src/components/shared/btn-primary.tsx

```
@@ -0,0 +1,34 @@
+        //     "--btn-bg": bgColor,      
+        //     "--btn-text": textColor,   
+        // } as React.CSSProperties
+        //     } 
+      className={cn(
+        "inline-flex items-center justify-between gap-2 px-4 py-2.5 bg-maroon-600 dark:bg-softpink-200"
+        // bgColor,
+        // textColor
+      )}
+    >
+      <span className={cn("text-white text-base leading-none dark:text-zinc-800")}>{text}</span>
+      <ArrowRight className="w-4 h-4 inline-block align-middle" color="#ffffff" />
```

ESLint

[error] className value must use cn() function. Example: className={cn("...")}

(local-rules/require-cn-classname)